target_snowflake/__init__.py

```diff
@@ -189,8 +181,18 @@ def persist_lines(config, lines):
             else:
                 stream_to_sync[stream] = DbSync(config, o)
 
-            stream_to_sync[stream].create_schema_if_not_exists(table_columns_cache)
-            stream_to_sync[stream].sync_table(table_columns_cache)
+            try:
+                stream_to_sync[stream].create_schema_if_not_exists()
+                stream_to_sync[stream].sync_table()
+            except Exception as e:
+                logger.error("""
+                    Cannot sync table structure in Snowflake schema: {} .
+                    Try to delete {}.COLUMNS table to reset information_schema cache. Maybe it's outdated.
```

Comment:
instead of asking to refresh cache manually, why not do that here automatically?

Reply:
We can do it automatically but we need to restart the tap to take effect.

What should be the error code if this happens? Failed or Success?